instrumentation/jdbc/README.md

```diff
@@ -1,5 +1,6 @@
 # Settings for the JDBC instrumentation
 
-| System property                                         | Type    | Default | Description                            |
-|---------------------------------------------------------|---------|---------|----------------------------------------|
-| `otel.instrumentation.jdbc.statement-sanitizer.enabled` | Boolean | `true`  | Enables the DB statement sanitization. |
+| System property                                         | Type    | Default | Description                                           |
+|---------------------------------------------------------|---------|---------|-------------------------------------------------------|
+| `otel.instrumentation.jdbc.statement-sanitizer.enabled` | Boolean | `true`  | Enables the DB statement sanitization.                |
+| `otel.instrumentation.jdbc.experimental.txn.enabled`    | Boolean | `false` | Enables the experimental transaction instrumentation. |
```

Comment:
Replace `txn` with `transaction`. I think the description could be changed to indicate that this option enables creation of spans for `COMMIT` and `ROLLBACK` operations. The current description doesn't really reveal what this instrumentation does.